Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -1202,34 +1199,7 @@ IPCCommandResult ES::LaunchBC(const IOCtlVRequest& request)
 
 void ES::ResetAfterLaunch(const u64 ios_to_load) const
```

Comment:
Note: I'm not replacing ResetAfterLaunch with a simple call to Reload(), because there will be things to add to this function in a future PR.